src/main/java/com/onarandombox/MultiverseCore/commands/CoordinatesCommand.java

```diff
@@ -0,0 +1,61 @@
+package com.onarandombox.MultiverseCore.commands;
+
+import co.aikar.commands.BukkitCommandIssuer;
+import co.aikar.commands.annotation.CommandAlias;
+import co.aikar.commands.annotation.CommandPermission;
+import co.aikar.commands.annotation.Description;
+import co.aikar.commands.annotation.Subcommand;
+import com.onarandombox.MultiverseCore.api.LocationManipulation;
+import com.onarandombox.MultiverseCore.api.MVWorld;
+import com.onarandombox.MultiverseCore.api.MVWorldManager;
+import com.onarandombox.MultiverseCore.commandtools.MVCommandManager;
+import com.onarandombox.MultiverseCore.commandtools.MultiverseCommand;
+import com.onarandombox.MultiverseCore.utils.MVCorei18n;
+import jakarta.inject.Inject;
+import org.bukkit.ChatColor;
+import org.bukkit.World;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+import org.jvnet.hk2.annotations.Service;
+
+@Service
+@CommandAlias("mv")
+public class CoordinatesCommand extends MultiverseCommand {
+
+    private final MVWorldManager worldManager;
+    private final LocationManipulation locationManipulation;
+
+    @Inject
+    public CoordinatesCommand(@NotNull MVCommandManager commandManager, @NotNull MVWorldManager worldManager, @NotNull LocationManipulation locationManipulation) {
+        super(commandManager);
+
+        this.worldManager = worldManager;
+        this.locationManipulation = locationManipulation;
+    }
+
+    @Subcommand("coordinates|coord|coords|co")
+    @CommandPermission("multiverse.core.coordinates")
```

Comment:
I think for now perms node just follow MV4

```suggestion
    @CommandPermission("multiverse.core.coord")
```